java/Nikiforov-Oleksandr/src/com/zoolatech/lecture4/tasks/_2/BinarySearchTree.java

```diff
@@ -1,43 +1,43 @@
 package com.zoolatech.lecture4.tasks._2;
 
 public class BinarySearchTree {
-    static class Node {
+    private class Node {
```

Comment:
the class should remain `static`, since it doesn't need access to any internal state of a parent class